backend/src/test/resources/application-test.yml

```diff
@@ -0,0 +1,6 @@
+spring:
+  datasource:
+    username: postgres
+    password: postgres
```

Comment:
Is this a real password in plaintext?

Reply:
This is the password we use for testing purposes only as you can see it's `application-test.yaml`, this password is created by `docker-compose` when new database container is initialized